src/stories/Library/material-search/MaterialSearch.stories.tsx

```diff
@@ -5,12 +5,29 @@ import MaterialSearch from "./MaterialSearch";
 export default {
   title: "Library / Material Search",
   component: MaterialSearch,
-  argTypes: {},
+  argTypes: {
+    displayWorkError: {
+      defaultValue: false,
+      control: {
+        type: "boolean",
+      },
+    },
+    displayMaterialTypeError: {
+      defaultValue: false,
+      control: {
+        type: "boolean",
+      },
+    },
```

Comment:
Consider making this an enum or the like with 3 states - no error, work error, material type error that can be toggled in a single control.

Reply:
Yeah thats cleaner